misc/netgen/meta.yaml

```diff
@@ -1,12 +1,12 @@
 # Use `conda-build-prepare` before building for a better version string.
-{% set version = '%s_%04i_%s'|format(GIT_DESCRIBE_TAG, GIT_DESCRIBE_NUMBER|int, GIT_DESCRIBE_HASH or 'gUNKNOWN') %}
+{% set version = '%s_%04i_%s'|format(GIT_DESCRIBE_TAG | replace('mpw-one-a', '1.5.219'), GIT_DESCRIBE_NUMBER|int, GIT_DESCRIBE_HASH or 'gUNKNOWN') %}
```

Comment:
ah yes, I got this wrong. I think I was confused because conda-prepare complained about that tag, so I just assumed it matched `git_rev`.

Reply:
That's the error I'm getting:
```
==> /usr/bin/git describe --tags --dirty <==

mpw-one-a-142-gbfb01e0

==> /usr/bin/git status <==

On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean

Adding in variants from internal_defaults
INFO:conda_build.variants:Adding in variants from internal_defaults
Error: bad character '-' in package/version: mpw-one-a_0142_gbfb01e0
```

Reply:
So that's starting to become interesting:
```
🍡 git describe
1.5.118-218-gbfb01e0
🍙 git describe --tags
mpw-one-a-142-gbfb01e0
🍊 git describe --tags --debug
describe HEAD
No exact match on refs or tags, searching to describe
finished search at 395f857a824e6a8f4b9f91ad7472646cf2f43969
 lightweight      142 mpw-one-a
 lightweight      217 1.5.119
 annotated        218 1.5.118
traversed 219 commits
mpw-one-a-142-gbfb01e0
🍙 git tag -l --sort=-taggerdate | head -n1
1.5.219
```

Reply:
Not sure what's up with `master` but `netgen-1.5` branch doesn't seem to have the same issue:
```
🍙 git describe
1.5.219
🍡 git describe --tags
1.5.219
🍡 git describe --tags --debug
describe HEAD
1.5.219
🍙 git tag -l --sort=taggerdate | head -n1
1.5.119
```

Reply:
I think I get it, `master` doesn't have all the tags in it:
```
🍊 git tag --merged netgen-1.5 | wc -l
183
🍙 git tag --merged master | wc -l
83
🍡 git branch --contains 1.5.219
* netgen-1.5
```

/cc @RTimothyEdwards FYI